src/cpp/include/openvino/genai/visual_language/pipeline.hpp

```diff
@@ -90,16 +90,18 @@ class OPENVINO_GENAI_EXPORTS VLMPipeline {
     /// uint8 RGB images with [NHWC] or [HWC] layout.
     /// @param prompt A prompt to respond to.
     /// @param images Images to be prepended to a prompt.
+    /// @param video Video frames to be prepended to a prompt.
     /// @param generation_config A config to follow for text generation.
     /// @param streamer A streamer to acquire intermediate result.
     /// @return A string generated by a model.
     /// chat_template will be applied to the prompt, run pipe.set_chat_template(custom_chat_template) to update it.
     /// To disable it for non-chat mode, please, use custom_chat_template eq "" or set generation_config.apply_chat_template to false.
     VLMDecodedResults generate(
         const std::string& prompt,
-        const std::vector<ov::Tensor>& rgbs,
+        const std::vector<ov::Tensor>& images,
         const GenerationConfig& generation_config,
-        const StreamerVariant& streamer
+        const StreamerVariant& streamer,
+        const bool& is_video = false
```

Comment:
This seems to be incomplete. `video` property is introduced. `/// @param video Video frames to be prepended to a prompt.` is added. But the API still uses `is_video`.

To preserve backward compatibility, add new `generate()` and `add_request()` overloads with `video` video arg instead of inserting `video` in the middle of the arg list of existing method

Reply:
Yes, my initial idea was to add new `generate` function with video arg, but the video and image have same data type. 
Of course, `const bool& is_video = false` itself also is ugly. As you said, I add **new** `generate `function as follow:
```
    VLMDecodedResults generate(
        const std::string& prompt,
        const std::vector<ov::Tensor>& images,
        const std::vector<ov::Tensor>& video,
        const GenerationConfig& generation_config,
        const StreamerVariant& streamer
    );
```
Or I can take generation_option param from https://github.com/openvinotoolkit/openvino.genai/pull/2758/files#diff-526ed28503beb4d7c74482dce7fcd569660ee63d5c26c35bec243e90ba968e05R166, pass video via {{"video", video}}.

What do you think?